tensorflow/core/ops/array_ops.cc

```diff
@@ -2266,6 +2266,8 @@ size(t) ==> 12
 
 namespace {
 
+// This SliceHelper processes the output shape of the `slice`
+// when the tensor of `sizes` is available.
 template <typename T>
 Status SliceHelper(InferenceContext* c, ShapeHandle begin_value,
```

Comment:
SliceHelper can also be removed now?

Reply:
@cwhipkey This one is in the original code when tensor of size is available. I think it is needed when `size == -1` as it will check the tensor value.The previous added `SliceHelper` (different function signature) in this PR has already been removed.

Reply:
ah, thanks, sorry about that.